CDP4CrossViewEditor.Tests/ViewModels/ThingSelectorViewModelTestFixture.cs

```diff
@@ -0,0 +1,130 @@
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4CrossViewEditor.Tests.ViewModels
+{
+    using System;
+
+    using CDP4Common.CommonData;
+    using CDP4Common.EngineeringModelData;
+    using CDP4Common.SiteDirectoryData;
+
+    using CDP4CrossViewEditor.ViewModels;
+
+    using CDP4Dal;
+
+    using Moq;
+
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Suite of tests for the <see cref="ThingSelectorViewModel"/> class
+    /// </summary>
+    [TestFixture]
+    public class ThingSelectorViewModelTestFixture
+    {
+        /// <summary>
+        /// The current session associated <see cref="Assembler"></see>
+        /// </summary>
+        private Assembler assembler;
+
+        /// <summary>
+        /// The current assembler <see cref="Uri"/>
+        /// </summary>
+        private Uri uri;
+
+        /// <summary>
+        /// Mock <see cref="ISession"/>
+        /// </summary>
+        private Mock<ISession> session;
+
+        /// <summary>
+        /// Current iteration used for test
+        /// </summary>
+        private Iteration iteration;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.uri = new Uri("http://www.rheageoup.com");
+            this.assembler = new Assembler(this.uri);
+            this.session = new Mock<ISession>();
+            this.session.Setup(x => x.Assembler).Returns(this.assembler);
+
+            this.iteration = this.CreateIteration();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+        }
+
+        [Test]
+        public void VerifyThatPropertiesAreSet()
+        {
+            var viewModel = new ThingSelectorViewModel(this.iteration, ClassKind.ElementBase);
+
+            Assert.AreEqual(this.iteration, viewModel.Iteration);
+            Assert.AreEqual(0, viewModel.SourceThingList.Count);
+            Assert.AreEqual(0, viewModel.TargetThingList.Count);
+        }
+
+        [Test]
+        public void VerifyThatCommandsWorks()
+        {
+            var viewModel = new ThingSelectorViewModel(this.iteration, ClassKind.ElementBase);
+
+            Assert.DoesNotThrow(() => viewModel.MoveItemsToSource.Execute(null));
+            Assert.DoesNotThrow(() => viewModel.MoveItemsToTarget.Execute(null));
+            Assert.DoesNotThrow(() => viewModel.MoveItemsUp.Execute(null));
+            Assert.DoesNotThrow(() => viewModel.MoveItemsDown.Execute(null));
+            Assert.DoesNotThrow(() => viewModel.ClearItems.Execute(null));
+        }
+
+        [Test]
+        public void VerifyBindings()
+        {
+            var viewModel = new ThingSelectorViewModel(this.iteration, ClassKind.ElementBase);
+            viewModel.BindData();
+            viewModel = new ThingSelectorViewModel(this.iteration, ClassKind.ParameterBase);
+            viewModel.BindData();
+        }
+
+        private Iteration CreateIteration()
```

Comment:
Could we inline this in the `Setup` method? Seems unnecessary to have a separate method that is used only once for something this simple.